src/main/java/com/iota/iri/service/snapshot/impl/LocalSnapshotManagerImpl.java

```diff
@@ -296,4 +297,15 @@ public void addPruningConditions(PruningCondition... conditions) {
             this.pruningConditions = ArrayUtils.addAll(this.pruningConditions, conditions);
         }
     }
+
+    @Override
+    public int maxSnapshotPruningMilestone() throws TransactionPruningException {
+        int max = 0;
+        for (PruningCondition condition: pruningConditions) {
+            if(condition.getSnapshotPruningMilestone() > max){
+                max = condition.getSnapshotPruningMilestone();
```

Comment:
This should be the smallest milestone no? As we only prune the one that is lowest possible (to prevent pruning too much)

Reply:
Ah. Yes